oxidation/libparsec_python/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 name = "libparsec._libparsec"
 
 [lib]
-name = "libparsec"
+name = "libparsec_py"
```

Comment:
Don't we need to update the import statement in python code too ?

Reply:
`oxidation` tests pass, so 🤷

Reply:
> `python oxidation` tests pass, so 🤷

Can you test with a build from scratch, just to be extra sure the cache is not playing us here

Reply:
Also I'm curious: what the name of the .so file that endup in the final libparsec python package? Is it still `libparsec.so` ? (Hence maturin is doing a rename when installing the lib, which would explain why we don't need to change the python code to do the import)

Reply:
after doing `pip install oxidation/libparsec_python` `maturin` generate the following wheel file `target/wheels/libparsec-1.0.0-cp37-abi3-linux_x86_64.whl`

Reply:
Ok si the change you do is only relevant for the build cache which is great ^^

Reply:
> > `python oxidation` tests pass, so :shrug:
> 
> Can you test with a build from scratch, just to be extra sure the cache is not playing us here

```
$ poetry shell
$ cargo clean
$ ls target
ls: cannot access 'target': No such file or directory
$ pip install oxidation/libparsec_python
[...]
Successfully installed libparsec-1.0.0
[...]
$ ls target/{release,wheels}
target/release:
build  deps  examples  incremental  liblibparsec_python.d  liblibparsec_python.so

target/wheels:
libparsec-1.0.0-cp37-abi3-linux_x86_64.whl
$ python -m pytest tests --runrust -q
[...]
1352 passed, 395 skipped, 4 xfailed in 29.46s
```